include/odp/api/spec/comp.h

```diff
@@ -0,0 +1,863 @@
+/* Copyright (c) 2014-2018, Linaro Limited
+ * All rights reserved.
+ *
+ * SPDX-License-Identifier:     BSD-3-Clause
+ */
+
+/**
+ * @file
+ *
+ * ODP Compression
+ */
+
+#ifndef ODP_API_COMP_H_
+#define ODP_API_COMP_H_
+
+#include <odp/visibility_begin.h>
+#include <odp/api/support.h>
+#include <odp/api/packet.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/** @defgroup odp_compression ODP COMP
+ *  ODP Compression is an API set to do compression+hash or decompression+hash
+ *  operations on data. Hash is calculated on plaintext.
```

Comment:
Comp/decomp without hash is also supported. This chapter gives an impression that hash is always calculated.

" ... API to do compression or decompression operations on data. Hash calculation may be combined with comp/decomp operations..."

Reply:
I have updated the documentation.